src/main/java/car/app/entity/Car.java

```diff
@@ -1,22 +1,50 @@
 package car.app.entity;
 
 import java.sql.Timestamp;
+import java.util.Date;
 
+import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
 
 @Entity
+@Table(name = "car.car")
 public class Car {
 
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id")
 	private int id;
+
+	@Column(name = "name")
+	@NotNull(message = "NAME CANNOT BE NULL")
 	private String name;
+	
+	@Column(name = "registration")
 	private Timestamp Registration;
+
+	@Column(name = "country")
```

Comment:
if a @notNull validation in the entity is included, a not null property should also be included within @column so the table in postgres contains the validation too